Scripts/Triplets/x64-macos-plasma.cmake

```diff
@@ -12,3 +12,7 @@ if(PORT STREQUAL cairo)
     set(VCPKG_LIBRARY_LINKAGE dynamic)
     set(VCPKG_BUILD_TYPE release)
 endif()
+
+if(PORT STREQUAL python3)
+    set(VCPKG_CONFIGURE_MAKE_OPTIONS "LDFLAGS_NODIST=-rpath ${CURRENT_INSTALLED_DIR}/lib")
```

Comment:
The Python documentation [says](https://docs.python.org/3/using/configure.html#envvar-CONFIGURE_LDFLAGS_NODIST) that this should be `CONFIGURE_LDFLAGS_NODIST`

Reply:
`CONFIGURE_LDFLAGS_NODIST` does not seem to work. Build fails when I switch to that argument.

Reply:
What is the error?

Reply:
It's the same error this is trying to fix in the Python Cairo module. It's as if no configure arguments are applied at all.